{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "python310-Django4-4.2.17-1.1 on GA media",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "These are all security issues fixed in the python310-Django4-4.2.17-1.1 package on the GA media of openSUSE Tumbleweed.",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "openSUSE-Tumbleweed-2024-14565",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/opensuse-su-2024_14565-1.json"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2024-53907 page",
				"url": "https://www.suse.com/security/cve/CVE-2024-53907/"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2024-53908 page",
				"url": "https://www.suse.com/security/cve/CVE-2024-53908/"
			}
		],
		"title": "python310-Django4-4.2.17-1.1 on GA media",
		"tracking": {
			"current_release_date": "2024-12-09T00:00:00Z",
			"generator": {
				"date": "2024-12-09T00:00:00Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "openSUSE-SU-2024:14565-1",
			"initial_release_date": "2024-12-09T00:00:00Z",
			"revision_history": [
				{
					"date": "2024-12-09T00:00:00Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python310-Django4-4.2.17-1.1.aarch64",
								"product": {
									"name": "python310-Django4-4.2.17-1.1.aarch64",
									"product_id": "python310-Django4-4.2.17-1.1.aarch64"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django4-4.2.17-1.1.aarch64",
								"product": {
									"name": "python311-Django4-4.2.17-1.1.aarch64",
									"product_id": "python311-Django4-4.2.17-1.1.aarch64"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django4-4.2.17-1.1.aarch64",
								"product": {
									"name": "python312-Django4-4.2.17-1.1.aarch64",
									"product_id": "python312-Django4-4.2.17-1.1.aarch64"
								}
							},
							{
								"category": "product_version",
								"name": "python313-Django4-4.2.17-1.1.aarch64",
								"product": {
									"name": "python313-Django4-4.2.17-1.1.aarch64",
									"product_id": "python313-Django4-4.2.17-1.1.aarch64"
								}
							}
						],
						"category": "architecture",
						"name": "aarch64"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python310-Django4-4.2.17-1.1.ppc64le",
								"product": {
									"name": "python310-Django4-4.2.17-1.1.ppc64le",
									"product_id": "python310-Django4-4.2.17-1.1.ppc64le"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django4-4.2.17-1.1.ppc64le",
								"product": {
									"name": "python311-Django4-4.2.17-1.1.ppc64le",
									"product_id": "python311-Django4-4.2.17-1.1.ppc64le"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django4-4.2.17-1.1.ppc64le",
								"product": {
									"name": "python312-Django4-4.2.17-1.1.ppc64le",
									"product_id": "python312-Django4-4.2.17-1.1.ppc64le"
								}
							},
							{
								"category": "product_version",
								"name": "python313-Django4-4.2.17-1.1.ppc64le",
								"product": {
									"name": "python313-Django4-4.2.17-1.1.ppc64le",
									"product_id": "python313-Django4-4.2.17-1.1.ppc64le"
								}
							}
						],
						"category": "architecture",
						"name": "ppc64le"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python310-Django4-4.2.17-1.1.s390x",
								"product": {
									"name": "python310-Django4-4.2.17-1.1.s390x",
									"product_id": "python310-Django4-4.2.17-1.1.s390x"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django4-4.2.17-1.1.s390x",
								"product": {
									"name": "python311-Django4-4.2.17-1.1.s390x",
									"product_id": "python311-Django4-4.2.17-1.1.s390x"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django4-4.2.17-1.1.s390x",
								"product": {
									"name": "python312-Django4-4.2.17-1.1.s390x",
									"product_id": "python312-Django4-4.2.17-1.1.s390x"
								}
							},
							{
								"category": "product_version",
								"name": "python313-Django4-4.2.17-1.1.s390x",
								"product": {
									"name": "python313-Django4-4.2.17-1.1.s390x",
									"product_id": "python313-Django4-4.2.17-1.1.s390x"
								}
							}
						],
						"category": "architecture",
						"name": "s390x"
					},
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python310-Django4-4.2.17-1.1.x86_64",
								"product": {
									"name": "python310-Django4-4.2.17-1.1.x86_64",
									"product_id": "python310-Django4-4.2.17-1.1.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "python311-Django4-4.2.17-1.1.x86_64",
								"product": {
									"name": "python311-Django4-4.2.17-1.1.x86_64",
									"product_id": "python311-Django4-4.2.17-1.1.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "python312-Django4-4.2.17-1.1.x86_64",
								"product": {
									"name": "python312-Django4-4.2.17-1.1.x86_64",
									"product_id": "python312-Django4-4.2.17-1.1.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "python313-Django4-4.2.17-1.1.x86_64",
								"product": {
									"name": "python313-Django4-4.2.17-1.1.x86_64",
									"product_id": "python313-Django4-4.2.17-1.1.x86_64"
								}
							}
						],
						"category": "architecture",
						"name": "x86_64"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "openSUSE Tumbleweed",
								"product": {
									"name": "openSUSE Tumbleweed",
									"product_id": "openSUSE Tumbleweed",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:tumbleweed"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django4-4.2.17-1.1.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.aarch64"
				},
				"product_reference": "python310-Django4-4.2.17-1.1.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django4-4.2.17-1.1.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.ppc64le"
				},
				"product_reference": "python310-Django4-4.2.17-1.1.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django4-4.2.17-1.1.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.s390x"
				},
				"product_reference": "python310-Django4-4.2.17-1.1.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python310-Django4-4.2.17-1.1.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.x86_64"
				},
				"product_reference": "python310-Django4-4.2.17-1.1.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django4-4.2.17-1.1.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.aarch64"
				},
				"product_reference": "python311-Django4-4.2.17-1.1.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django4-4.2.17-1.1.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.ppc64le"
				},
				"product_reference": "python311-Django4-4.2.17-1.1.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django4-4.2.17-1.1.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.s390x"
				},
				"product_reference": "python311-Django4-4.2.17-1.1.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python311-Django4-4.2.17-1.1.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.x86_64"
				},
				"product_reference": "python311-Django4-4.2.17-1.1.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django4-4.2.17-1.1.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.aarch64"
				},
				"product_reference": "python312-Django4-4.2.17-1.1.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django4-4.2.17-1.1.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.ppc64le"
				},
				"product_reference": "python312-Django4-4.2.17-1.1.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django4-4.2.17-1.1.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.s390x"
				},
				"product_reference": "python312-Django4-4.2.17-1.1.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python312-Django4-4.2.17-1.1.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.x86_64"
				},
				"product_reference": "python312-Django4-4.2.17-1.1.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python313-Django4-4.2.17-1.1.aarch64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.aarch64"
				},
				"product_reference": "python313-Django4-4.2.17-1.1.aarch64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python313-Django4-4.2.17-1.1.ppc64le as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.ppc64le"
				},
				"product_reference": "python313-Django4-4.2.17-1.1.ppc64le",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python313-Django4-4.2.17-1.1.s390x as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.s390x"
				},
				"product_reference": "python313-Django4-4.2.17-1.1.s390x",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python313-Django4-4.2.17-1.1.x86_64 as component of openSUSE Tumbleweed",
					"product_id": "openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.x86_64"
				},
				"product_reference": "python313-Django4-4.2.17-1.1.x86_64",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2024-53907",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2024-53907"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An issue was discovered in Django 5.1 before 5.1.4, 5.0 before 5.0.10, and 4.2 before 4.2.17. The strip_tags() method and striptags template filter are subject to a potential denial-of-service attack via certain inputs containing large sequences of nested incomplete HTML entities.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.aarch64",
					"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.ppc64le",
					"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.s390x",
					"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.x86_64",
					"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.aarch64",
					"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.ppc64le",
					"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.s390x",
					"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.x86_64",
					"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.aarch64",
					"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.ppc64le",
					"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.s390x",
					"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.x86_64",
					"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.aarch64",
					"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.ppc64le",
					"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.s390x",
					"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2024-53907",
					"url": "https://www.suse.com/security/cve/CVE-2024-53907"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1234232 for CVE-2024-53907",
					"url": "https://bugzilla.suse.com/1234232"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 6.5,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-12-09T00:00:00Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2024-53907"
		},
		{
			"cve": "CVE-2024-53908",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2024-53908"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An issue was discovered in Django 5.1 before 5.1.4, 5.0 before 5.0.10, and 4.2 before 4.2.17. Direct usage of the django.db.models.fields.json.HasKey lookup, when an Oracle database is used, is subject to SQL injection if untrusted data is used as an lhs value. (Applications that use the jsonfield.has_key lookup via __ are unaffected.)",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.aarch64",
					"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.ppc64le",
					"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.s390x",
					"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.x86_64",
					"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.aarch64",
					"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.ppc64le",
					"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.s390x",
					"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.x86_64",
					"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.aarch64",
					"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.ppc64le",
					"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.s390x",
					"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.x86_64",
					"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.aarch64",
					"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.ppc64le",
					"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.s390x",
					"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2024-53908",
					"url": "https://www.suse.com/security/cve/CVE-2024-53908"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1234231 for CVE-2024-53908",
					"url": "https://bugzilla.suse.com/1234231"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 9.1,
						"baseSeverity": "CRITICAL",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:N",
						"version": "3.1"
					},
					"products": [
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python310-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python311-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python312-Django4-4.2.17-1.1.x86_64",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.aarch64",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.ppc64le",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.s390x",
						"openSUSE Tumbleweed:python313-Django4-4.2.17-1.1.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-12-09T00:00:00Z",
					"details": "critical"
				}
			],
			"title": "CVE-2024-53908"
		}
	]
}
